packages/p2p/src/components/my-ads/quick-add-modal.jsx

```diff
@@ -73,9 +73,23 @@ const QuickAddModal = ({ advert }) => {
         }
     };
 
+    const hideModal = should_close_all_modals => {
```

Comment:
The function  name `hideModal` doesn't make sense if it's accepting true or false, let's rename this to `setShouldCloseAllModals`